src/Components/Aspire.Orleans.Client/Aspire.Orleans.Client.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Following the naming pattern, this (and the server) should have `Microsoft` in the name. For example:

`Aspire.Microsoft.Orleans.Client`